src/hooks/useWalletCloudBackup.js

```diff
@@ -15,53 +16,93 @@ import logger from 'logger';
 export default function useWalletCloudBackup() {
   const dispatch = useDispatch();
   const { wallets } = useWallets();
-  const { goBack } = useNavigation();
 
   const walletCloudBackup = useCallback(
-    async ({ walletId, password, latestBackup, onError, onSuccess }) => {
-      try {
-        dispatch(setIsWalletLoading(walletLoadingStates.BACKING_UP_WALLET));
+    async ({
+      createBackupFileIfNeeded,
+      handleNoLatestBackup,
+      handlePasswordNotFound,
+      latestBackup,
```

Comment:
I think we should remove this one from the parameters and just get it directly from the useWallets hooks.